test/e2e/clusterctl_upgrade_test.go

```diff
@@ -143,7 +143,7 @@ var _ = Describe("When testing clusterctl upgrades (v1.3=>current)", func() {
 			// try to deploy the latest version of our test-extension from docker.yaml.
 			InitWithRuntimeExtensionProviders: []string{},
 			InitWithProvidersContract:         "v1beta1",
-			InitWithKubernetesVersion:         "v1.26.0",
+			InitWithKubernetesVersion:         "v1.27.0",
```

Comment:
```suggestion
			InitWithKubernetesVersion:         "v1.26.0",
```

The maximum support version for v1.3 will remain v1.26, so let's keep this on v1.26

Reply:
Oh yeah, thx, great catch, updated now.